clientd3d/bspload.c

```diff
@@ -60,7 +60,7 @@ Bool BSPRooFileLoad(char *fname, room_type *room)
    int node_pos, wall_pos, sidedef_pos, sector_pos, offset_adjust;
    file_node f;
 
-   if (!MappedFileOpenCopy(fname, &f))
+   if (!CliMappedFileOpenRead(fname, &f))

```

Comment:
Weird, I wonder why it was originally done this way.

It looks like this is the only use of MappedFileOpenCopy, so you can delete that function now.